src/actions/actions.ts

```diff
@@ -2112,8 +2112,8 @@ class CommandGoToDefinition extends BaseCommand {
 
 @RegisterAction
 class CommandInsertAtFirstCharacter extends BaseCommand {
-  modes = [ModeName.Normal];
-  mustBeFirstKey = true;
+  modes = [ModeName.Normal, ModeName.Visual];
+  mustBeFirstKey = false;
```

Comment:
@johnfn  I actually think it is wrong to be first key, in Vim if you have a line of text, and in the middle press 4At<Esc> it will append 4 t's to the end. With this set to true that definitely can't happen right?